README.md

```diff
@@ -200,3 +200,14 @@ A string template that will be interpolated on the client using two route
 params, `repositoryId` and `activityId`, into a preview URL for each activity.
 Example:
 `https://my.url.com/#/repository/{repositoryId}/activity/{activityId}/preview`
+
+## EXTENSIONS
+
+Tailor support creation of custom content elements and custom containers. Content
+elements are pieces of learning that a student would consume (text, image, etc.)
```

Comment:
I prefer avoiding learning terminology because that might not be the only ecosystem to use it.